.github/workflows/tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Tests
+on:
+  pull_request:
+  push:
+    branches:
+    - main
+    - gha
+  schedule:
+    # Weekly.
+    - cron: "0 0 * * 0"
+
+jobs:
+  track-config:
+    name: Check track configuration
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Fetch
+        uses: exercism/github-actions/configlet-ci@main
+      - name: Lint
+        run: configlet lint
```

Comment:
You could, if you'd like, remove this bit as there is already a [separate workflow](https://github.com/exercism/d/blob/main/.github/workflows/configlet.yml) that does this.